ext/bcmath/libbcmath/src/doaddsub.c

```diff
@@ -33,6 +33,12 @@
 #include "private.h"
 #include <stddef.h>
 
+/* This will be 0x01010101 for 32-bit and 0x0101010101010101 */
```

Comment:
Missing words?
```suggestion
/* This will be 0x01010101 for 32-bit and 0x0101010101010101 for 64-bit. */
```

Reply:
This is also just a copy, but it seems to have been missing from the original. Fixed.